test/_elasticsearch.sh

```diff
@@ -5,7 +5,7 @@ export SHIPPABLE_ES_PORT=9200;
 export SHIPPABLE_ES_BINARY="/usr/local/bin/elasticsearch";
 #export SHIPPABLE_ES_USER=elasticsearch
 export SHIPPABLE_ES_TMP=/usr/local/elasticsearch/tmp
-export SHIPPABLE_ES_CMD="ES_JAVA_OPTS=\"-Djna.tmpdir=$SHIPPABLE_ES_TMP -Djava.io.tmpdir=$SHIPPABLE_ES_TMP\" ES_BINARY -Ecluster.name=$SHIPPABLE_ES_CLUSTER_NAME";
+export SHIPPABLE_ES_CMD="/usr/local/bin/elasticsearch -Ecluster.name=$SHIPPABLE_ES_CLUSTER_NAME";
```

Comment:
can we use the env `SHIPPABLE_ES_BINARY` instead of path

Reply:
ok, updating